apollo-runtime/src/main/java/com/apollographql/apollo/cache/normalized/CacheControl.java

```diff
@@ -1,8 +1,12 @@
 package com.apollographql.apollo.cache.normalized;
 
+import com.apollographql.apollo.cache.CacheHeaders;
+
 /**
- * CacheControl class represents the cache controlling strategies for reading GraphQL responses back from the
- * {@link NormalizedCache}.
+ * {@link CacheControl} represents strategies for defining what precedence reading a GraphQL responses back
+ * from the {@link NormalizedCache}.
+ *
+ * To control how a cache reads a response see {@link CacheHeaders}.
```

Comment:
Is it just for reading `To control how a cache reads`. I thought CacheHeaders is used for writing as well